File: src/plugin/command/owner/SendVarCommandPlugin.java
package plugin.command.owner;

import org.redrune.game.content.plugin.type.CommandPlugin;
import org.redrune.game.entity.actor.player.Player;

/**
 * @author Tyluur <itstyluur@gmail.com>
 * @since 2019-01-29
 */
public class SendVarCommandPlugin extends CommandPlugin {
	
	@Override
	public void handle(Player player, String[] args, boolean console, boolean clientCommand) {
		int id = intParamOrDefault(args, 1, 0);
		int value = intParamOrDefault(args, 2, 0);
		player.getVarManager().sendVar(id, value);
	}
	
	@Override
	public String[] identifiers() {
		return arguments("sendvar");
	}
}

File: src/main/kotlin/plugin/command/owner/SendVarCommandPlugin.kt
package plugin.command.owner

import org.redrune.game.content.plugin.type.CommandPlugin
import org.redrune.game.entity.actor.player.Player

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 2019-01-29
 */
class SendVarCommandPlugin : CommandPlugin() {
    override fun handle(player: Player, args: Array<String>, console: Boolean, clientCommand: Boolean) {
        val id = intParamOrDefault(args, 1, 0)
        val value = intParamOrDefault(args, 2, 0)
        player.varManager.sendVar(id, value)
    }

    override fun identifiers(): Array<String> {
        return arguments("sendvar")
    }
}